### PR TITLE
DOC: Sphinx 3 compatibility

### LINKED ISSUE #3205: Sphinx 3 issues
Tracking thread for Sphinx 3 upgrade.

* [ ] `plot_workflow` directive is failing:
   ```
    Creating file /tmp/src/nipype/doc/users/examples/smoothing_comparison.rst.

    Exception occurred:
      File "/usr/local/lib/python3.7/site-packages/nipype/sphinxext/plot_workflow.py", line 256, in setup
        app.add_directive("workflow", wf_directive, True, (0, 2, False), **options)
    TypeError: add_directive() got an unexpected keyword argument 'alt'
   ```

### PR DESCRIPTION
**Pull request by effigies**
## Summary
The `:workflow:` directive uses a deprecated API that was removed in Sphinx 3.

Fixes #3205.

## List of changes proposed in this PR (pull-request)

* Refactors the workflow directive.
* Adds https://github.com/marketplace/actions/run-circleci-artifacts-redirector to make reviewing documentation changes more straightforward.

## Acknowledgment

- [x] \(Mandatory\) I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

**Comment by effigies** [comment]
It looks like we don't use this directive anywhere, so there are no docs to check. Using poldracklab/fmriprep#2092 to test.

**Comment by satra** [comment]
i think we were using it in nipype workflows which has moved to niflows.

**Comment by effigies** [comment]
I don't think we're building the docs there, yet.

Functionality is preserved using the new API with Sphinx 2: https://13143-53175327-gh.circle-artifacts.com//0/tmp/src/fmriprep/docs/_build/html/workflows.html

**Comment by effigies** [comment]
Also with Sphinx 3: https://13149-53175327-gh.circle-artifacts.com//0/tmp/src/fmriprep/docs/_build/html/workflows.html

File: doc/conf.py [hunk 1]
```diff
@@ -189,143 +217,154 @@
...
-apidoc_module_dir = '../nipype'
-apidoc_output_dir = 'api/generated'
+apidoc_module_dir = "../nipype"
+apidoc_output_dir = "api/generated"
 apidoc_excluded_paths = [
-    '*/tests/*', 'tests/*',
-    'external/*',
-    'fixes/*',
-    'scripts/*',
-    'sphinxext/*',
```
Comment (effigies) [inline doc/conf.py:316]: I removed this line, and then `black` had to have this file.

We're now at least exercising the code in doc builds again, even if it won't show up in coverage. I'm going to merge.

**Pull request merged**